test/tests/data/search/a.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>hello</title>
+  </head>
+  <body>
+    <p>mjktkiuewf</p>
```

Comment:
I wouldn't worry too much about the couple seconds that it takes to reset the DB. The additional data may make the test more robust, but we can't really count on it, since we have no idea how populated the database is at that point (e.g. some test right before this ends up resetting the DB). So it really gives us no additional confidence. Overall, I still prefer legibility and I still think that once `resetDB` is working robustly, these tests should reset the DB. Not a big deal right now though.